CoroutineTemplate/build.gradle

```
@@ -85,11 +85,11 @@ detekt {
     reports {
         xml {
             enabled = true
-            destination = file("build/reports/detekt.xml")
+            destination = file("build/reports/detekt/detekt-checkstyle.xml")
```

I think `detekt.xml` is okay enough, `checkstyle` word makes me think about `checkstyle` plugin instead, while `detekt` and `checkstyle` are different 💭 
```suggestion
            destination = file("build/reports/detekt/detekt.xml")
```

---

or
```suggestion
            destination = file("build/reports/detekt/detekt-result.xml")
```